src/lib/share/__tests__/generateShareLink.js

```diff
@@ -42,7 +42,9 @@ describe('generateShareLink', () => {
     const params = {
       key: 'value',
     }
-    let paramsBase64 = Buffer.from(JSON.stringify(params)).toString('base64')
+    let paramsBase64 = Buffer.from(JSON.stringify(params))
```

Comment:
Maybe create some util method and reuse it where needed.
i.e.
```
const paramsBase64 = parseToBase64(params)
```

Reply:
+1, that's a good idea as this code repeats too much